test/e2e/gateway/tokens_only.go

```diff
@@ -0,0 +1,128 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gateway
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/firefly-common/pkg/fftypes"
+	"github.com/hyperledger/firefly/pkg/core"
+	"github.com/hyperledger/firefly/test/e2e"
+	"github.com/hyperledger/firefly/test/e2e/client"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type TokensOnlyTestSuite struct {
+	suite.Suite
+	testState *testState
+	connector string
+	db        string
+	key       string
+}
+
+func (suite *TokensOnlyTestSuite) SetupSuite() {
+	suite.testState = beforeE2ETest(suite.T())
+	stack := e2e.ReadStack(suite.T())
+	stackState := e2e.ReadStackState(suite.T())
+
+	suite.connector = stack.TokenProviders[0]
+	suite.db = stack.Database
+	account := stackState.Accounts[0].(map[string]interface{})
+	suite.key = account["address"].(string)
```

Comment:
Do we need to extract this rather than just using the default key?

Reply:
Looking through the code, is there any better way to get the `defaultkey`? I don't see it used anywhere.

Reply:
You should be able to just leave `key` blank and asset manager will always use the default.